### PR TITLE
tag: add page

### PR DESCRIPTION
**Pull request by a-well-lit-room**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/tag.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# tag
+
+> Edit tags on Mac OS X files (10.9 Mavericks and above).
+> More information: <https://github.com/jdberry/tag/>.
+
+- Add a tag to a file:
```
Comment (spageektti) [inline pages/osx/tag.md:6]:
```suggestion
- Add tags to a file:
```

File: pages/osx/tag.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`tag --add {{tagname1,tagname2}} {{path/to/file}}`
```
Comment (spageektti) [inline pages/osx/tag.md:8]:
```suggestion
`tag --add {{tagname1,tagname2,...}} {{path/to/file}}`
```

File: pages/osx/tag.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Remove a tag:
+
+`tag --remove {{tagname}} {{path/to/file}`
+
+- Remove all tags from a file:
+
+`tag --remove \* {{file}}`
```
Comment (spageektti) [inline pages/osx/tag.md:16]:
```suggestion
`tag --remove \* {{path/to/file}}`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thank you so much for your contribution! Welcome to tldr-pages!

**Comment by a-well-lit-room** [comment]
I incorporated all of your suggested edits. Thank you!

File: pages/osx/tag.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`tag --remove {{tagname}} {{path/to/file}`
+
+- Remove all tags from a file:
+
+`tag --remove \* {{path/to/file}}`
+
+- Show all files with a given tag:
+
+`tag --match {{tagname}}`
```
Comment (kbdharun) [inline pages/osx/tag.md:20]:
```suggestion
`tag --add {{tag_name1,tag_name2,...}} {{path/to/file}}`

- Remove a tag:

`tag --remove {{tag_name}} {{path/to/file}`

- Remove all tags from a file:

`tag --remove \* {{path/to/file}}`

- Show all files with a given tag:

`tag --match {{tag_name}}`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution.  I would suggest using the snake case for the page's "tag name" placeholder.

**Ref**. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming

After this suggestion, the PR is GTG.

**Review by gutjuri — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**